cloudferrylib/utils/cmd_cfg.py

```diff
@@ -27,5 +28,8 @@
 ssh_cmd_port = base_ssh_cmd("-oStrictHostKeyChecking=no -p %s %s '%s'")
 dd_cmd_of = BC("dd bs=%s of=%s")
 dd_cmd_if = BC("dd bs=%s if=%s")
+dd_full = BC('dd if=%s of=%s bs=%s count=%s seek=%sM')
 gunzip_cmd = BC("gunzip")
-gzip_cmd = BC("gzip -%s -c %s")
\ No newline at end of file
+gzip_cmd = BC("gzip -%s -c %s")
+scp_cmd = BC('scp %s %s@%s:%s %s')
+rm_cmd = BC('rm -rf %s')
```

Comment:
You don't really need `-r` in `rm`, you're only removing one file. Let's modify it to `rm -f %s`.